doc/FAQ.md

```diff
@@ -176,16 +176,18 @@ Example: Friendica Support
 
 Friendica supports [Mastodon API](help/API-Mastodon) and [Twitter API | gnusocial](help/api).
 This means you can use some of the Mastodon and Twitter clients for Friendica.
-The available features are client specific and may differ.
+The available features are client specific and may differ. Clients dedicated to Friendica are marked in **bold**.
```

Comment:
Please add each sentence on a separate line. Display won't be changed but future changes in individual sentences are easier to review.
```suggestion
The available features are client specific and may differ.
Clients dedicated to Friendica are marked in **bold**.
```